docs/plugins/gatsby-theme-flow/src/components/rate-this.js

```diff
@@ -0,0 +1,60 @@
+import styled from "@emotion/styled";
+
+import { useMixpanel } from "gatsby-plugin-mixpanel";
+
+import React, { useState, useEffect } from "react";
+
+const StarButton = styled.button({
+  backgroundColor: "transparent",
+  border: "none",
+  outline: "none",
+  cursor: "pointer",
+  marginLeft: "-6px",
+  "&.on": {
+    color: "#ffb400"
+  },
+  "&.off": {
+    color: "#ccc"
+  }
+});
+
+const StarRating = ({ pageInfo }) => {
+  const [rating, setRating] = useState(0);
+  const [hover, setHover] = useState(0);
+  const [hasRated, setHasRated] = useState(false);
+  const mixpanel = useMixpanel();
+
+  useEffect(() => {
+    if (rating) {
+      mixpanel.track("Page rating", {
+        rating,
+        pageInfo
+      });
+      setHasRated(true);
+    }
+  }, [rating]);
+
+  return (
+    <div>
+      <div>{hasRated ? "Rating submitted, thank you!" : "Rate this page"}</div>
```

Comment:
Feel free to say no to this but was thinking we could also include the following link to google form for any qualitative feedback once the rating is submitted: https://docs.google.com/forms/d/e/1FAIpQLSeytoATNQ9UqA24VxGrf4OdESzBKvHC4pMKD5uCEGCcOqn4fg/viewform?entry.389052155=https://docs.onflow.org

Note the `entry.389052155=https://docs.onflow.org` replace with the pageInfo.path for prefill to work.